app/presenters/archived_tag_presenter.rb

```diff
@@ -3,18 +3,34 @@ def initialize(tag)
     @tag = tag
   end
 
+  def draft?
```

Comment:
Perhaps we could use `delegate` like in the `TagPresenter` for the methods that call `@tag`.